### PR TITLE
MISC-363 update 2022.13.2

### PR DESCRIPTION
**Pull request by tpetillot**
chore: update theme version to v0.0.0-20220222162046-58693df56c0f
---
 
chore: update simulation general step documentation
---

**Modification:**
General step now include package configuration, and have a dropdown for classe name

**Ref:** MISC-363
 
chore: update reports documentation

---

**Modification:**
Update screen with new design

chore: update package documentation, closes MISC-363

---

**Modification:**
* Update screen without global team on creation
* Update simulation step

**Ref:** MISC-363

### CODE REVIEW

File: content/reference/user/reports/index.md [hunk 1]
```diff
@@ -61,6 +53,14 @@ You can change the time range with control buttons or by selecting a region on t
 
 {{< img src="timeline.png" alt="Timeline" >}}
 
+## Assertions
+
+The image below shows the status of the simulation (Ongoing, successful, timeout...).
```
Comment (slandelle) [inline content/reference/user/reports/index.md:58]: Either use capitals or not, but don't mix

**Review by slandelle — APPROVED** [review]

File: content/reference/user/simulations/index.md [hunk 2]
```diff
@@ -50,6 +50,10 @@ If you want specific properties for a simulation, you will be allowed to ignore
 
 ## Creating a simulation
 
+Creating a simulation require to have an uploaded package.
+
+See how to [generate a package]({{< ref "../package_gen" >}}) and [create it]({{< ref "../package_conf/#creation" >}}) on Gatling Enterprise Cloud.
```
[the remark this answers is not in the document]
Comment (Isammoc) [reply content/reference/user/simulations/index.md:55]: `You have to upload a package first`

Active voice

File: content/reference/user/reports/index.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ The Reports page consists of:
 
 This bar is a combination of buttons including:
 
-- **Start / Stop**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)
-- **Generate public link**: To create a [public link]({{< ref "#public-links" >}}) that can be viewed by anyone it is sent to.
-- **Export**: To [export]({{< ref "#export-pdf" >}}) reports
+- **Start / Stop**: Start a new run of the simulation, or stop the ongoing run
+- **Generate public link**: Create a [public link]({{< ref "#public-links" >}}) that can be viewed by anyone it is sent to.
```
Comment (Isammoc) [inline content/reference/user/reports/index.md:34]:
```suggestion
- **Share**: Create a [public link]({{< ref "#public-links" >}}) to share this reports to anyone you want.
```

**Review by Isammoc — COMMENTED** [review]
Are `create-simulation3` and `create-simulation4` mixed?

Note: the 3rd step in Simulation is **_Pools_** but the image shows now the JVM options...

Perhaps should we have more rigid file name naming convention? (numbers are bad)

**Review by Isammoc — APPROVED** [review]

**Pull request merged**